src/main/kotlin/de/bund/digitalservice/useid/tracking/matomo/MatomoTrackingService.kt

```diff
@@ -37,20 +36,15 @@ class MatomoTrackingService(trackingProperties: TrackingProperties, private val
         return url
     }
 
-    private fun sendEvent(e: MatomoEvent) {
-        Mono.fromCallable {
-            constructEventURL(e)
-        }.zipWhen {
-            webRequests.POST(it)
-        }.map {
-            val status = it.t2.statusCode
-            if (status == HttpStatus.OK) {
-                log.info("$status, successfully tracked: ${it.t1}")
-            } else {
-                log.error("$status, tracking failed for: ${it.t1}")
-            }
-        }.subscribe()
-    }
+    @EventListener
+    @Async
+    fun sendEvent(e: MatomoEvent) {
+        val eventUrl = constructEventURL(e)
 
-    override fun onApplicationEvent(e: MatomoEvent) = sendEvent(e)
+        if (webRequests.POST(eventUrl)) {
+            log.info("Successfully tracked: $eventUrl")
+        } else {
+            log.error("Tracking failed for: $eventUrl")
```

Comment:
I guess we will improve our logging strategy soon anyway.
But until then I suggest we use following strings for consistency:
```suggestion
        if (webRequests.POST(eventUrl)) {
            log.info("Tracking ok: $eventUrl")
        } else {
            log.error("Tracking fail: $eventUrl")
```

Reply:
Done